factory/.env

```diff
@@ -0,0 +1,38 @@
+## This env represents the latest versions cyrpess supports. If you wish to push an older docker image,
+## please modify circle ci to release a one off branch instead of merging old versions to this file in master.
```

Comment:
Are there instructions for this? I'm not totally sure what this means. I guess the intent is to not do this that often anyways.

Reply:
I don't really have instructions for it, largely it would come down to modifying the filters on the push jobs.

https://github.com/cypress-io/cypress-docker-images/blob/matth/docker-factory/circle.yml#L271-L274

one could setup a one off branch and create at a PR to go into it.

Reply:
I had the same thought here. I'd say it's worth including this bit as a _for example..._ one-liner to give even subtle guidance here since some users may not know how to interpret this.